xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/hierarchy_macros.vm

```diff
@@ -146,7 +146,10 @@
       #set ($breadCrumbParameters = "$breadCrumbParameters data-${option.toLowerCase()}='$escapetool.xml($options.get($option))'")
     #end
   #end
-  <ol #if("$!options.id" != '')id="$options.id"#end class="$breadCrumbClassNames" $breadCrumbParameters><li
+  ## Set as a navigation role because it contains mostly links.
+  <nav aria-label="#if("$!options.id" != '')$options.id#end $breadCrumbClassNames">
```

Comment:
I find it strange to use the id and class names as the value for the ``aria-label``. I thought the practice is to either use a human readable text with ``aria-label`` (which is not the case here) or use ``aria-labelledby`` to target an existing element.

Reply:
Thanks! I'll look for a solution and fix this soon. 
I really messed up this PR state, so if it's okay I'll close it and start a new fresh PR from https://github.com/Sereza7/xwiki-platform/tree/XWIKI-20696
Right now because of my mess ups, it's almost impossible to review the PR anymore and I'm pretty sure I can't fix this myself :/

Reply:
Using the id makes sure we have a unique aria-label, which is what we're aiming for. I'm working on an implementation of a label parameter. However, default hierarchies (without a label) will have the same label, which could be problematic in some cases. 
> If a page includes more than one navigation landmark, each should have a unique label.

https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/navigation.html

Reply:
> Right now because of my mess ups, it's almost impossible to review the PR anymore and I'm pretty sure I can't fix this myself :/

Something like the following should set the PR in a correct state:
```bash
$ git fetch
$ git rebase origin/master ## possibly fixing some conflicts
$ git push NAME_OF_REMOTE NAME_OF_LOCAL_BRANCH:NAME_OF_REMOTE_BRANCH --force-with-lease
```

Reply:
Okay, thanks!

Reply:
@manuelleduc Unfortunately, I can't reopen this PR, I created https://github.com/xwiki/xwiki-platform/pull/2133 instead and just after that saw your message.
This helped me "save" [this other PR](https://github.com/xwiki/xwiki-platform/pull/2107) though, thanks again!